.github/workflows/check.yml

```diff
@@ -7,9 +7,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+        with:
+          submodules: true
```

Comment:
The `examples/wasm-rustc` submodule should probably not be cloned. Maybe instead add an explicit `git submodule update --init test/wasi-testsuite`?

Reply:
Nice catch!